defos/src/defos_linux.cpp

```diff
@@ -102,6 +105,9 @@ void defos_init()
 
     current_cursor = NULL;
     memset(default_cursors, 0, DEFOS_CURSOR_INTMAX * sizeof(CustomCursor*));
+
+	// Get enter/exit cursor events
+	XSelectInput(disp, win, EnterWindowMask | LeaveWindowMask);
```

Comment:
From https://tronche.com/gui/x/xlib/event-handling/XSelectInput.html:

> Setting the event-mask attribute of a window overrides any previous call for the same window but not for other clients

Also, this http://read.pudn.com/downloads182/doc/850380/xlib.pdf seems to hint that you should first read the event mask and then add to it. That should be safer.